akka-http-testkit/src/main/scala/akka/http/javadsl/testkit/RouteTest.scala

```diff
@@ -33,7 +34,7 @@ abstract class RouteTest extends AllDirectives with WSTestRequestBuilding {
   implicit def materializer: Materializer
   implicit def executionContext: ExecutionContextExecutor = system.dispatcher
 
-  protected def awaitDuration: FiniteDuration = 3.seconds
+  protected def awaitDuration: FiniteDuration = new JavaTestKit(system).dilated(1.seconds).asInstanceOf[FiniteDuration];
```

Comment:
Sure, I would keep original setting. 
ktoso mentioned
> We do however depend on 2.4 nowadays since we want to still be compatible with it, thus do not want to leak in 2.5 only API usage.
Original post - #1151